site2/docs/concepts-messaging.md

```diff
@@ -176,9 +176,11 @@ Topic name component | Description
 
 A namespace is a logical nomenclature within a tenant. A tenant can create multiple namespaces via the [admin API](admin-api-namespaces.md#create). For instance, a tenant with different applications can create a separate namespace for each application. A namespace allows the application to create and manage a hierarchy of topics. The topic `my-tenant/app1` is a namespace for the application `app1` for `my-tenant`. You can create any number of [topics](#topics) under the namespace.
 
-## Subscription modes
+## Subscriptions
 
-A subscription is a named configuration rule that determines how messages are delivered to consumers. There are four available subscription modes in Pulsar: [exclusive](#exclusive), [shared](#shared), [failover](#failover), and [key_shared](#key_shared). These modes are illustrated in the figure below.
+A subscription is a named configuration rule that determines how messages are delivered to consumers.  Each consumer may use a unique subscription name exclusive only to it, and then each consumer on the topic would receive all messages independent of other consumers (fan-out pub-sub messaging).  Alternatively multiple consumers may use the same subscription name in either a shared or failover mode (message queuing). 
```

Comment:
I am not sure this clarification here is clear enough. topic-subscription already indicates that messages are fan-out to different subscriptions. The clarification seems to be indicating it is one way or another. I would suggest just adding a tip at the end of this section.

```
> If you want to achieve traditional "fanout pubsub messaging" among consumers, you can make each consumer ...
> If you want to achieve "message queuing" among cnosumer, you can ...
```

Reply:
Updated with your suggestion of making it a separate note and added a bullet point that it isn't just an either-or proposition as you can do both simultaneously.